rustls-libssl/tests/runner.rs

```diff
@@ -31,6 +31,9 @@ use std::{fs, net, thread, time};
 #[test]
 #[ignore]
 fn client_unauthenticated() {
+    let port = choose_port();
+    let port_str = &format!("{port}");
```

Comment:
nit: I think this feels a little more natural as:

```rust
let port_str = &port.to_string();
```

Alternatively, maybe `choose_port()` should return `(u16, String)`? It seems like in all cases we end up also needing to `format!()` or `to_string()` a String as well.

Reply:
> Alternatively, maybe `choose_port()` should return `(u16, String)`?

yeah that seems reasonable for test code.